kafka-quickstart/processor/pom.xml

```diff
@@ -34,6 +34,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
+        </dependency>
```

Comment:
I think this change is unrelated? I don't mind adding it if it's not a mistake.

Reply:
Yes, I have the same question

Reply:
hum.... sorry, I didn't stage that but the github cli did. Let me remove it.